components/server/src/auth/login-completion-handler.ts

```diff
@@ -48,9 +47,6 @@ export class LoginCompletionHandler {
                 });
             });
         } catch (err) {
-            // Clean up the session & avoid loops
-            await AuthFlow.clear(request.session);
```

Comment:
I wonder if we want to do some cleanup in all the places we did `AuthFlow.clear` before. But I cannot think of any... we should always use the hostUrl to craft redirects (like here), so we should be fine as well. :+1:

Reply:
Yeah, I had a look at refs to `.clear()` but couldn't find anything else to drop/adjust.